troughs_and_waves/gaussian/tests/test_gaussian_factory.py

```diff
@@ -0,0 +1,30 @@
+#  (C) Copyright 2010-2020 Enthought, Inc., Austin, TX
```

Comment:
I think we can remove this file now, the factory tests look like they are already in place in ``test_gassian.py`